### PR TITLE
Add qk100 keyboard v2 and v3 files

### PR DESCRIPTION
**Pull request by owlab-git**
## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/21782

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by owlab-git** [comment]
Please check, thanks.

File: v3/qwertykeys/qk100/qk100_ansi.json [hunk 1]
```diff
@@ -0,0 +1,419 @@
+{
+  "name": "qk100 ansi",
+  "vendorId": "0x4F53",
+  "productId": "0x3030",
+  "keycodes": ["qmk_lighting"],
+  "menus": [
```
Comment (Xelus22) [inline v3/qwertykeys/qk100/qk100_ansi.json:6]:
```suggestion
  "menus": [ "qmk_rgb_matrix" ],
```

you can replace the entire RGB block with this.

also you have rgb MATRIX enabled, not rgb LIGHT in your qmk firmware.

File: v3/qwertykeys/qk100/qk100_solder.json [hunk 2]
```diff
@@ -0,0 +1,368 @@
+{
+  "name": "qk100 Solder",
+  "vendorId": "0x4F53",
+  "productId": "0x3053",
+  "keycodes": ["qmk_lighting"],
+  "menus": [
```
Comment (Xelus22) [inline v3/qwertykeys/qk100/qk100_solder.json:6]:
```suggestion
  "menus": [ "qmk_rgblight" ],
```

File: v3/qwertykeys/qk100/qk100_solder.json [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+  },
+  "layouts": {
+    "labels": [
+      "2U Home",
+      "Split Backspace",
+      "ISO Enter",
+      "Split Left Shift"
+    ],
+    "keymap": [
+      [
```
Comment (Xelus22) [inline v3/qwertykeys/qk100/qk100_solder.json:99]: image

suggest moving ISO out a bit (both files)
image

**Review by Xelus22 — CHANGES_REQUESTED** [review]
Please remove src/ folder ones. those are ONLY for legacy boards

**Pull request merged**